packit_service/worker/handlers/koji.py

```diff
@@ -313,6 +315,23 @@ def report(self, description: str, commit_status: BaseCommitStatus, url: str):
             url=url,
         )
 
+        if (
+            not KojiOpenScanHubHelper.osh_disabled()
+            and self.db_project_event.type == ProjectEventModelType.pull_request
+            and self.build.target == "fedora-rawhide-x86_64"
+        ):
+            try:
+                KojiOpenScanHubHelper(
+                    koji_build_helper=self.koji_build_helper,
```

Comment:
I think `"KojiTaskReportDownstreamHandler" has no attribute "koji_build_helper"` is quite self-explanatory. You'll have to get it in some other way (if it is actually needed).

Reply:
I have tried to fix it in the code by deriving `KojiTaskReportDownstreamHandler` from `KojiBuildHandler`.